tests/loaders/test_acquire.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+from dissect.target.loaders.tar import TarLoader
+from tests._utils import absolute_path
+
+if TYPE_CHECKING:
+    from pytest_benchmark.fixture import BenchmarkFixture
+
+    from dissect.target import Target
+    from dissect.target.loader import Loader
+
+
+@pytest.mark.parametrize(
+    ("archive", "loader"),
+    [
+        ("_data/loaders/tar/test-windows-fs-c-relative.tar", TarLoader),
+    ],
+)
+@pytest.mark.benchmark
+def test_benchmark(benchmark: BenchmarkFixture, target_default: Target, archive: str, loader: type[Loader]) -> None:
+    file = absolute_path(archive)
+
+    benchmark(lambda: loader(file).map(target_default))
```

Comment:
```suggestion
def test_benchmark(benchmark: BenchmarkFixture, archive: str, loader: type[Loader]) -> None:
    file = absolute_path(archive)

    benchmark(lambda: loader(file).map(Target()))
```
I think this is a slightly more accurate test.